app/views/pets/results.html.erb

```diff
@@ -1,70 +1,71 @@
-<div class="panel panel-info">
-	<div class="panel-heading">
-		<h3 class="panel-title">Sign up for PetAlerts for this Search</h3>
-	</div>
-	<div class="panel-body">
-		<p>
-			You can receive alerts by email or text message when a new pet is added that matches your search conditions. We will not share your contact information or spam you. You can cancel an alert any time by <%= link_to 'clicking here', unsubscribe_url %>.
+<div class= "container">
+	<div class= "subscription-form">
+		<h2 class="call-to-action centered">Get Alerts for this Search</h2>
+		<p class= "centered">
+			Receive alerts by email or text message when a new pet matching this search is posted.
 		</p>
 
 		<%= form_for @subscription, url: subscribe_url, html: { method: 'post' } do |f| %>
-		<div class='form-group col-xs-4'>
-			<%= f.label 'species' %>
-			<%= f.text_field 'species', class: 'form-control', readonly: true %>
+
+		<div class='subscription-readonly-input'>
+			<%= f.label 'species', 'species:' %>
+			<%= f.text_field 'species', class: 'form-control', readonly: true, placeholder: 'view all' %>
 		</div>
-		<div class='form-group col-xs-4'>
-			<%= f.label 'gender' %>
-			<%= f.text_field 'gender', class: 'form-control', readonly: true %>
+		<div class='subscription-readonly-input'>
+			<%= f.label 'gender', 'gender:' %>
+			<%= f.text_field 'gender', class: 'form-control', readonly: true, placeholder: 'view all' %>
 		</div>
-		<div class='form-group col-xs-4'>
-			<%= f.label 'found_since' %>
-			<%= f.text_field 'found_since', class: 'form-control', readonly: true %>
+		<div class='subscription-readonly-input'>
+			<%= f.label 'found', 'found:' %>
+			<%= f.text_field 'found_since', class: 'form-control', readonly: true, disabled: true, placeholder: 'view all' %>
```

Comment:
@Tallys We can't use `disabled: true` because then the value won't be submitted as part of the subscription. We should be disabling the javascript action by removing whatever class is binding the js UI to our form element.